src/cli.js

```diff
@@ -25,7 +25,8 @@ import { exit } from 'process'
 function parseCommand(rawArgs) {
   checkNodeVersion()
 
-  const args = rawArgs.slice(2)
+  // should update paths converted by POSIX in case of process.env.MSYSTEM is present
+  const args = rawArgs.slice(2).map((arg) => arg.replace(/(.*)\/Git\//i, '/'))
```

Comment:
My concerns here are:
1. What if the current folder does not end with "Git"
2. What if the SAS Drive folder contains the word "Git"

Can we somehow extract that renegade local directory, and use it in the regex?